perf_dashboard/perf_dashboard/settings.py

```diff
@@ -39,7 +40,13 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['localhost', 'perf.dashboard.qualistio.org']
+ALLOWED_HOSTS = [
+    'localhost',
+    'perf.dashboard.qualistio.org',
```

Comment:
I'm wondering if we are using 'perf.dashboard.istio.io' do we still need this 'perf.dashboard.qualistio.org'?

Reply:
It depends on whether we want to continue to serve from `perf.dashboard.qualistio.org` for a short period of time for migration (and forward via a cname record)?

Reply:
yeah, keep it there for now, after the 'perf.dashboard.istio.io' is stable, we no longer need it.